### ISSUE TITLE
Results on Office31

### ISSUE THREAD

**Issue by shreejalt** [zysymu/AdaMatch-pytorch#2]
Hi @zysymu 
Thanks for your contribution. 
I tried using the Office31 dataset using Adamatch to get some results. But I was unable to reproduce it and the accuracy was very low.

Can you let me know the hyperparameters to change or have you tried training on the Office31 dataset?
